src/mayank/AssignmentFrequency.java

```diff
@@ -0,0 +1,20 @@
+package mayank;
+
+public class AssignmentFrequency{
+	void frequency(String input){
+		int count=0;
+		
+		for(int i=input.length()-1;i>=0;i--){
+		
+			if(input.charAt(i) == 't'){
```

Comment:
use generic process, your code will only work for character 't', you have to pass any character as a parameter and your code will print frequency of it